apps/src/applab/applab.js

```diff
@@ -557,6 +558,9 @@ Applab.init = function (config) {
   config.runButtonClickWrapper = runButtonClickWrapper;
 
   Applab.channelId = config.channel;
+  Applab.firebaseName = config.firebaseName;
+  Applab.firebaseAuthToken = config.firebaseAuthToken;
```

Comment:
Would be nice if we didn't have to stick these on Applab. What if we passed them into FirebaseStorage, i.e. the next line would become:
```
Applab.storage = window.dashboard.project.getUseFirebase() ? FirebaseStorage(config.firebasename, config.firebaseAuthToken) : AppStorage;
```
And would return us a FirebaseStorage with the name/auth token captured.